main/res/values/strings.xml

```diff
@@ -1839,7 +1831,7 @@
 
     <plurals name="days_ago">
         <item quantity="zero">%d days ago</item>
-        <item quantity="one">yesterday</item>
+        <item quantity="one">%d day ago</item>
```

Comment:
Why don't keep yesterday?

Reply:
Ignore me, found the reason in https://github.com/cgeo/cgeo/issues/10155#issue-831002685